# Request
float64 z_value     # target z value
---
# Response
bool success
string message